src/structures/interfaces/TextBasedChannel.js

```diff
@@ -297,7 +297,7 @@ class TextBasedChannel {
     if (Array.isArray(messages) || messages instanceof Collection) {
       let messageIds = messages instanceof Collection ? [...messages.keys()] : messages.map(m => m.id ?? m);
       if (filterOld) {
-        messageIds = messageIds.filter(id => Date.now() - SnowflakeUtil.deconstruct(id).timestamp < 1209600000);
+        messageIds = messageIds.filter(id => Date.now() - SnowflakeUtil.deconstruct(id).timestamp < 1_209_600_000);
```

Comment:
Not for this PR but we should probably move this magic number to a const someday 👀

Reply:
What even is this number?

Reply:
2 weeks IIRC

Reply:
Could probably make this a constant inside this function then, since it’s not used anywhere and wouldn’t be out of the scope of the PR iirc

Reply:
Nah. We good.